pkg/operation/botanist/component/vpa/admissioncontroller.go

```diff
@@ -208,6 +208,7 @@ func (v *vpa) reconcileAdmissionControllerDeployment(deployment *appsv1.Deployme
 						fmt.Sprintf("--port=%d", admissionControllerPort),
 						"--register-webhook=false",
 					},
+					LivenessProbe: newDefaultLivenessProbe(),
```

Comment:
Don't we need a readiness probe as well for the VPA admission controller as it is a component that also receives traffic?

Reply:
You're right, the admission controller doesn't have a readiness probe yet. As we had seen at least one occurrence where a VPA component was stuck, we wanted to integrate the already existing health check function as liveness to resolve this. If we have a good candidate for readiness we can also add this to the admission controller – maybe in an additional PR?

Reply:
Ok, let's postpone it to potentially other PR. I now read that it is not recommended to set the same configuration for both the probes - the liveness probe is much more "aggressive" -> it forces a container restarts and shadows the effect of a failing readiness probe.